src/client/editor/Editor.js

```diff
@@ -123,12 +124,12 @@ export default class Editor {
     this._ignoreSceneModification = false;
     this.signals.sceneGraphChanged.add(() => {
       if (this._ignoreSceneModification) return;
-      this.sceneInfo.modified = true;
+      this.sceneInfo.modified = this.sceneInfo.saved = true;
```

Comment:
Hmm, this is confusing. The scene is considered "saved" when it is modified? This can't be right.